tests/conftest.py

```diff
@@ -101,7 +101,8 @@ def factory(interface=constants.CEPHBLOCKPOOL):
             ).get(defaults.CEPHFILESYSTEM_NAME)
             ceph_pool_obj = OCS(**cfs)
         assert ceph_pool_obj, f"Failed to create {interface} pool"
-        instances.append(ceph_pool_obj)
+        if interface == constants.CEPHBLOCKPOOL:
```

Comment:
Why is this change needed?

Reply:
We want to delete the pool only if that's cephblockpool.
cephfilesystem shouldn't be removed

Reply:
@RazTamir this should be a different PR in itself, its like add on additional fix along with teardown factory PR :)

Reply:
fixed in https://github.com/red-hat-storage/ocs-ci/pull/572